src/components/App/App.js

```diff
@@ -15,15 +14,15 @@ function App() {
     const KEY = 'G3mzZaPrADRWteI33jNxbTRAM6TCT5NVzlc7za79';
 
     useEffect(() => {
-        console.log('running');
         const date = new Date();
-        const day = date.getDate();
-        const month = date.getMonth() + 1;
+        let day = date.getDate();
+        let month = date.getMonth() + 1;
         const year = date.getFullYear();
 
-        month < 10
-            ? setDate(`${year}-0${month}-${day}`)
-            : setDate(`${year}-${month}-${day}`);
+        month = month < 10 ? `0${month}` : month;
+        day = day < 10 ? `0${day}` : day;
+
+        setDate(`${year}-${month}-${day}`);
```

Comment:
if you change the code above this could be:

```suggestion
        setDate(`${year}-${month}-${day}`);
```